tests/examples/shims/awesome.lua

```diff
@@ -43,9 +43,26 @@ awesome._shim_fake_class = _shim_fake_class
 -- Avoid c.screen = acreen.focused() to be called, all tests will fail
 awesome.startup = true
 
+function awesome.xkb_get_group_names()
+    return "pc+us+inet(evdev)"
+end
+
+function awesome.xkb_get_layout_group()
+    return 0
+end
+
+
 function awesome.register_xproperty()
 end
 
+function awesome.xkb_get_group_names()
+    return "pc+us+inet(evdev)"
+end
+
+function awesome.xkb_get_layout_group()
+    return 0
+end
```

Comment:
> All the issues that I agreed from @psychon comments in all related PRs are addressed or voluntarily ignored.

Uhm, I'm not quite sure what "voluntarily ignored" means, so I will not repeat my comments on this code...
(Duplication here, `s.outputs` is still wrong, I stopped reading at this point)